### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by Donmark2k**
### Blog app: add devise
In this branch, I @Donmark2k , @aubinleyoung  and @Johnadibe implement the following;
- Build a web app that requires the user to log in.
- Use devise gem for authentication.

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <br>
+    <% end %>
+    <div class="actions">
+      <%= f.submit "Log in",  class:'user_input' %>
+    </div>
+  <% end %>
+  <div class="user_setting">
+    <%= render "devise/shared/links" %>
+  </div>
+</div>
```
Comment (CollinsTatang) [inline app/views/devise/sessions/new.html.erb]: Hi @Donmark2k 
- [x] Please style the registration and login page to match the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)
image

**Review by CollinsTatang — APPROVED** [review]

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Donmark2k, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
### **Notice** ⛔ 
Sorry for the above error, 

- [x] Please do the requested changes and submit for review again

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comment above._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (Donmark2k) [reply app/views/devise/sessions/new.html.erb]: @CollinsTatang, If I match this wireframe, how will a new user register?

Comment (CollinsTatang) [reply app/views/devise/sessions/new.html.erb]: Yeah @Donmark2k 
Can we have a call?

Comment (Donmark2k) [reply app/views/devise/sessions/new.html.erb]: @CollinsTatang , I have resolved it

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

- [OPTIONAL] It would be fine to add the sign-up link your root page for the new users to navigate easily.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**